vm_test.go

```diff
@@ -229,6 +230,7 @@ func BenchmarkEmptyLoop(b *testing.B) {
 	`
 	b.StopTimer()
 	vm := New()
+	vm.SetRateLimiter(rate.NewLimiter(rate.Limit(10_000_000), 250_000))
```

Comment:
```suggestion
	vm.SetRateLimiter(rate.NewLimiter(rate.Inf, 1))
```
The burst value shouldn't matter if you set the rate to `rate.Inf`